content/sigs/docs/index.adoc

```diff
@@ -11,13 +11,13 @@ tags:
   - docs_sig
   - documentation
 leads:
+- "kmartens27"
 - "markewaite"
 participants:
-- "oleg_nenashev"
-- "dheerajodha"
-- "kwhetstone"
+- "krisstern"
 - "stackscribe"
-- "zaycodes"
+- "vandit1604"
+
```

Comment:
```suggestion
- "gounthar"

```
If you don't mind.

Reply:
Thanks for catching that!  Fixed in 317b604cbf7ea774f852e0cf405348cdbc580967

Reply:
Thanks, Mark!